.prettierrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+	printWidth: 80,
+	useTabs: true,
+	tabWidth: 2,
+	overrides: [
+		{
+			files: "*.json",
+			options: {
+				useTabs: false
+			}
+		}
+	]
+};
```

Comment:
Use this config:
```
{
  "singleQuote": true,
  "trailingComma": "es5",
  "arrowParens": "always"
}
```

Reply:
Added,PTAL

Reply:
@evenstensberg I made a forced push to add `html` to prettier.

file: https://github.com/webpack/webpack-dev-server/blob/0185f6397e0edd0285fc2154d7e8fe94e9103f51/.prettierrc.js

Reply:
Looks like a regular config, got no opinion here. @evilebottnawi has worked with this more to have a better argument

Reply:
@evenstensberg  Oops, Sorry, I mistook the name of the destination 🙇